rasa/core/brokers/pika.py

```diff
@@ -276,12 +277,62 @@ def close(self) -> None:
 
     @property
     def rasa_environment(self) -> Optional[Text]:
+        """Get value of the `RASA_ENVIRONMENT` environment variable."""
         return os.environ.get("RASA_ENVIRONMENT")
 
+    @staticmethod
+    def _get_queues_from_args(
+        queue_arg: Union[List[Text], Tuple[Text], Text, None],
+        queues_arg: Union[List[Text], Tuple[Text], Text, None],
+    ) -> Union[List[Text], Tuple[Text]]:
+        """Get queues for this event broker.
+
+        Args:
+            queue_arg: Value of the supplied `queue` argument.
+            queues_arg: Value of the supplied `queues` argument.
+
+        Returns:
+            Queues this event broker publishes to.
+
+        """
+        if queue_arg:
+            raise_warning(
+                "Your Pika event broker config contains the deprecated `queue` key. "
+                "Please use the `queues` key instead.",
+                FutureWarning,
+                docs=DOCS_URL_PIKA_EVENT_BROKER,
+            )
+
+        if queues_arg and isinstance(queues_arg, (list, tuple)):
+            return queues_arg
+
+        elif queues_arg and isinstance(queues_arg, str):
+            raise_warning(
+                "Found a string value under the `queues` key of the Pika event broker "
+                "config. Please supply a list of queues under this key, even if it is "
+                "just a single one.",
+                docs=DOCS_URL_PIKA_EVENT_BROKER,
+            )
+            return [queues_arg]
+
+        if queue_arg and isinstance(queue_arg, str):
+            return [queue_arg]
```

Comment:
Probably is clearer if you put this after the `if queue_arg`, i.e. L 306

Reply:
no, because `queues` should take precedence if both are provided